### DISCUSSION TITLE
PASSPORT WITH SERVERLESS FUNCTIONS [VERCEL]

### DISCUSSION THREAD

**Discussion by Yacie** [jaredhanson/passport#809] [Help]
**Is it just me!?**

I cant get passport with its local & jwt strategies to work with serverless functions.
Previously, when working traditionally with MEVN stack I had no issues.

PLIZ ASSIST
**The best way to assist is to provide a simple working example of a serverless function with passport authentication**

**Comment by Yacie** [comment]
**The best way to assist is to provide a simple working example of a serverless function with passport authentication**

**Comment by llaenowyd** [comment]
+1 for this - @jaredhanson

Best I could find was [Implementing an Authentication Flow with Passport and Netlify Functions](https://markus.oberlehner.net/blog/implementing-an-authentication-flow-with-passport-and-netlify-functions/), where the solution is to spin up Express and adapt to the function with `serverless-http`.

Why not factor out the Oauth functionality and provide by default, but optionally, ExpressJS middleware adapter?  Is there something else that would better fit this use case?
